ui/components/portfolio-transaction-component.vue

```diff
@@ -287,7 +288,12 @@ const isValidTransaction = (
     (transaction.transactionType === 'BUY' || transaction.transactionType === 'SELL') &&
     typeof transaction.quantity === 'number' &&
     typeof transaction.price === 'number' &&
-    typeof transaction.transactionDate === 'string'
+    typeof transaction.transactionDate === 'string' &&
+    transaction.instrumentId !== undefined &&
+    transaction.transactionType !== undefined &&
+    transaction.quantity !== undefined &&
+    transaction.price !== undefined &&
+    transaction.transactionDate.trim() !== ''
```

Comment:
Please fix formatting by running this command: `npm run forma`

Reply:
on it